db/migrate/20170601233834_delete_available_column_from_game.rb

```diff
@@ -0,0 +1,4 @@
+class DeleteAvailableColumnFromGame < ActiveRecord::Migration[5.0]
+  def change
+  end
```

Comment:
That is weird.....looks like I haven't saved it before I pushed it :-)  
@mscoutermarsh it is now added in the migration and should be ready for review.

I thought while working in an Agile Team it is good practice to create new migration files rather than doing a rollback just to keep everything traceable ?